requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -18,86 +18,105 @@ The 'full' onboarding experience is delivered across some 'sub' campaigns, below
 This campaign triggers when a user completes the `user signed up` event for the first time, on either PostHog US or PostHog EU, provided they also have a valid email address.
```

Comment:
@joethreepwood I'm working on direct outreach to high ICP signups and want to make sure we're not overloading them with email, such that they will be more likely to ignore mine where I'm collecting info/offering to chat. 

This may take some back and forth (knowing the limits of orgs/permission levels/billing email visibility etc), but as a starting point, are you ok filtering out [this segment](https://fly.customer.io/workspaces/127208/journeys/segments/79/overview) (which is anyone with ICP > 10) from the auto emails? Filtered to the last 30 days it looks like this would cut ~320 users out. 

I have some other ideas for ways I'd further like to target these users (admin/owner, user count > 2) but there are some technical / logistical issues with some of that so I think this is a good place to start.

Reply:
I'm thinking I could review the list of users manually and figure out a way to re-enroll in onboarding emails those that it doesn't make sense to personally message.

Reply:
Hmm. We can do that, but I'm not 100% sure it is a good approach alone. Partly because I don't think emails are overwhelming (especially when spread over more than four weeks) and because if that _is_ the feeling I think better to tackle that. Open to feedback on this, basically. 

Beyond that, I want to make sure users get at least the welcome email (for encouraging usage) and the newsletter invite (for brand growth). So, there are two possible counter proposals I'd suggest, if you feel strongly about it. 

1. We could add a filter _after_ the newsletter invite (Step 7) which checks if the user has a high ICP. If the user has ICP > 10 then they skip straight to the end of the flow and do not get any subsequent emails or upsells. 

2. We could expand that filter so that, after Step 7, these users enter a different flow whereby they get the Beta Invite email and the Join The Community email spread over a 4 week period. I think these emails have value and would still free you up to reach out. 

Let me know what you think?

Reply:
I'm happy with 2 for now. 

Another option would be to modify these templates beyond a certain step, such that if they are high icp they get an additional snippet in them that offers my calendar, just to add a bit more of a personal touch. But I need to do more of these manually for now to get a better sense of who responds and what it is they want to talk to me about.